designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,18 @@
+# Beyond basic messaging: Designing long-running processes
+
+## Abstract
+
+Messaging in distributed systems offers a range of valuable benefits: it enables the design of decoupled components, where each component can evolve and scale individually. In addition, messaging improves the system’s fault tolerance but goes even further by opening the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s exactly the focus of this workshop.
+
+If you’ve been exposed to the basic building blocks of messaging and want to understand how to design, build, and test long-running processes in message-based distributed systems, this workshop is for you.
+
+Throughout this workshop, you’ll learn:
+
+- How to identify long-running processes in your domain
+- Effective techniques to help design complex business processes
+- What orchestration and choreography is, but more importantly, when and why to use each pattern
+- How to deal with actions that occur at different points in time
+- How to deal with out-of-order events seamlessly
+- How to test workflows and troubleshoot issues where it matters most: in production!
+
+To get the most out of this workshop, some basic messaging knowledge is expected, including understanding message brokers and messaging patterns like Request-Reply and Publish-Subscribe. Get ready to dive into design exercises in small groups and unlock the secrets of messaging's real-world superpowers!
```

Comment:
Before I go into a more in-depth review of the wordings etc. I have a more general question: While I was reading through the abstract, which I find generally already quite interesting/good, I was wondering what the target audience is. Is it more for Senior Devs, Lead Engineers, Architects and wannabees? It seems that this sentence hints at mostly design exercises which puts it probably at the level of lead devs, architects and wannabees and not so much at the level of devs. But then when it looked at 

>  build, and test long-running processes
and

> test workflows and troubleshoot issues

seems to be more in the area of devs. So I'm not sure what I should expect from the workshop. Will there be mostly concepts and design exercises or will I see some code and engineering practices or even build some stuff?

Reply:
I think given the breadth of exercises, there won't be any hands on coding (remember this is a one day workshop). Rather, there will be design exercises in group, and I will prefer code samples that can be referred to by the audience at a later point. But in case I would have more time than I know what to do with, these exercises could be done in class. However, I don't expect I'll get to them. If I grow this into a longer workshop, it could include both.

Honestly, I don't believe much in the split between devs and senior devs/architects/lead engineers. I believe any of these roles would benefit from the design exercises.

Reply:
For me it is not about a split but rather about me reading the abstract and not being entire sure what I could/should expect from the current wording because the highlighted parts seemed leave room for interpretation. Now that I better understand I might be able to make some concrete suggestions

Reply:
That would be great!